ballerina/sql_client.bal

```diff
@@ -96,13 +124,26 @@ public client class SQLClient {
         return resultStream;
     }
 
+    # Performs an SQL `SELECT` operation to read multiple records from the database when an advanced filter is provided.
+    #
+    # + filterClause - The filter query to be used (to be used in the SQL `WHERE` clauses)
```

Comment:
```suggestion
    # + filterClause - The filter query to be used in the SQL `WHERE` clauses
```

Reply:
Fixed in [83ba8ac](https://github.com/ballerina-platform/module-ballerina-persist/pull/29/commits/83ba8aca943aaaaffd88f25e28fe62a5c00c1ef6)